catroid/src/org/catrobat/catroid/formulaeditor/SensorHandler.java

```diff
@@ -177,10 +197,20 @@ public void onAccuracyChanged(Sensor arg0, int arg1) {
 	@Override
 	public void onSensorChanged(SensorEvent event) {
 		switch (event.sensor.getType()) {
+			case Sensor.TYPE_ACCELEROMETER:
+				accelerationXYZ = event.values.clone();
+				double norm_Of_g = Math.sqrt(accelerationXYZ[0] * accelerationXYZ[0]
```

Comment:
We use `camelCase` instead of `snake_case`. Also, is this your own code or did you get it from SO? If so, please leave a reference for the source.